src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor_Minimal.cs

```diff
@@ -47,7 +47,7 @@ protected override bool ShouldRestrictMinimallyQualifyLookupToNamespacesAndTypes
             var token = semanticModelOpt.SyntaxTree.GetRoot().FindToken(positionOpt);
             var startNode = token.Parent;
 
-            return SyntaxFacts.IsInNamespaceOrTypeContext(startNode as ExpressionSyntax);
+            return SyntaxFacts.IsInNamespaceOrTypeContext(startNode as ExpressionSyntax) || token.IsKind(SyntaxKind.NewKeyword);
```

Comment:
>|| token.IsKind(SyntaxKind.NewKeyword [](start = 89, length = 37)

I do not think this fix covers other scenarios where type arguments are involved. For example here is a clone of scenario from PropertyWithSameNameAsOtherType QuickInfo test :
```
namespace ConsoleApplication1
{
    class Program
    {
        public static List<Bar> Bar { get; set; }

        static void Main(string[] args)
        {
            Test<Bar>();
        }

        static void Test<T>() { }
    }

    class Bar
    {
    }
}
```

Hover over ```Test``` in ```Test<Bar>();```, it shows ```void Program.Test<ConsoleApplication1.Bar>()```. I would expect it to show ```void Program.Test<Bar>()```.